packages/hebao_v1/contracts/modules/security/InheritanceModule.sol

```diff
@@ -74,6 +74,7 @@ contract InheritanceModule is SecurityModule
         )
         external
         nonReentrant
+        onlyHaveEnoughGuardians(wallet)
```

Comment:
Isn't this a bad idea? Effectively the wallet is disabled if the owner is lost and there is only a single guardian.

I think the scenario of an owner (mobile wallet) + a single guardian (hardware wallet) is a very reasonable combination.

Reply:
This is an incorrect modification, Kongliang should change the recovery module, not this inheritance module.